### PR TITLE
fix #79548 nl.pepper.com

### LINKED ISSUE #79548: nl.pepper.com
### Issue URL (Annoyance)
 [https://nl.pepper.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnl.pepper.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Edge
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Dutch, <br/>EasyList Italy

### PR DESCRIPTION
**Pull request by wout-codes**
#79548

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 1]
```diff
@@ -2,6 +2,9 @@
 !######### White list. Fixing filtering errors #########
 !#######################################################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79548
+!+ PLATFORM(ios, ext_android_cb, ext_safari)
+nl.pepper.com#@#.cookies-message
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:7]: I don't see it.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -91,6 +91,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+nl.pepper.com#$#.popover-cover { display: none !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: better `.popover--oreo-message`, more specific

File: AnnoyancesFilter/sections/cookies_whitelist.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:7]: The site changed code.

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 2 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @Alex-302 Fixed.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**